data/coalition/coalition missions.txt

```diff
@@ -1393,3 +1393,283 @@ mission "Arachi Cartoonist 2"
+			`	"Enjoyed the games, have you, Captain?" The small box translates just as you turn around, seeing that Pairri has taken her own seat besides yours.`
+			choice
+				`	"Yeah, though I didn't have time to stop by your own match, sorry."`
+					goto notime
+				`	"Wait, why are you up here with the viewers?"`
+					goto why
+			
+			label notime
+			`	She mumbles something that the box doesn't pick up, and then says, "Cost me the match, impatience did," she says. "Still, much younger than most Kimek there, I am. Glad to have gotten so far, I feel."`
+				goto coop-game
+			
+			label prettygood
+			`	"Thank you. Glad to have gotten so far, I feel. Disappointing, defeat is, but try harder next year, I shall," she says."`
+				goto coop-game
+			
+			label why
+			`	"For the Kimek victors only, the cooperative match is reserved. Team up with Tumub and Ulmodi, my opponent will."`
+			
+			label coop-game
+			`	Once the mechanics are done, the ring gradually rises up in the air, and then fully takes off to leave the hangar. Out in the open, you overlook a massive track separated from the surrounding wilderness by a perfectly rectangular border of trees. Immediately under you the contestants stand near a starting line, divided into twenty groups of three. As you're brought closer to the teams, small monitors in front of you light up, and Pairri inputs a command for yours to focus on Tumub, Ulmodi, and, apparently their final teammate, the Kimek that beat Pairri in her match.`
+			`	The monitor also allows you to switch to a camera view of the entire track. It is a long, mostly flat stretch of land, leading to a rocky cliff wall on the far side that is cut to form a perfect right angle with the ground. Just beyond the top of the cliff is a river bank with many logs laid on the ground. The river itself is not terribly wide, but the current is so strong that the various Heliarch safety boats already waiting in the waters need to constantly adjust their position in order to avoid hitting the many rock outcroppings protruding from the water. Most significantly, though, as the river approaches the tree line, it drops off in a dangerous looking waterfall to resume its course on the lower ground.`
+			`	Pairri draws your attention back to the starting line, where Heliarch judges have the twenty teams line up. The Saryds, who seem to be working themselves up by hitting their hoofs repeatedly on the ground wait otherwise patiently. Meanwhile, their Kimek and Arach teammates pack and secure large bags and lengths of fabric in what appears to be a more conventional chariot. When the packing is complete and their teammates have boarded, the Saryds lift the yoke and strap themselves in before returning to their hoof stamping. As each team readies themselves, the Kimek and Arachi in the chariots pound on the side in time to the stamping and the teams begin to synch up, creating a uniform rhythm, like a giant heart beat.`
```

Comment:
```suggestion
			`	Pairri draws your attention back to the starting line, where Heliarch judges have the twenty teams line up. The Saryds seem to be working themselves up by hitting their hoofs repeatedly on the ground wait otherwise patiently. Meanwhile, their Kimek and Arach teammates pack and secure large bags and lengths of fabric in what appears to be a more conventional chariot. When the packing is complete and their teammates have boarded, the Saryds lift the yoke and strap themselves in before returning to their hoof stamping. As each team readies themselves, the Kimek and Arachi in the chariots pound on the side in time to the stamping and the teams begin to synch up, creating a uniform rhythm, like a giant heart beat.`
```